src/backend/marsha/e2e/test_lti.py

```diff
@@ -698,6 +698,7 @@ def test_lti_video_upload(page: Page, live_server: LiveServer):
 
     page.click("text=Create a video")
 
+    page.wait_for_selector("text=Video creation", state="detached")
```

Comment:
According to the commit description, we should wait a network idle event. Is it still the case ?

Reply:
Yes, commits messages are a bit messy for now.
I plan to rewrite them, and add more modifications.